src/server_manager/web_app/ui_components/outline-server-settings.js

```diff
@@ -359,16 +361,24 @@ Polymer({
   _validatePort: function(value) {
     const port = Number(value);
     const valid = !Number.isNaN(port) && port >= 1 && port <= 65535 && Number.isInteger(port);
-    return valid ? '' : this.localize('error-keys-port-bad-input');
+    if (!valid && this.localize) {
+      return this.localize('error-keys-port-bad-input');
+    }
+    return '';
   },
 
   _getShortName: getShortName,
+  _getCloudIcon: getCloudIcon,
+  _getCloudName: getCloudName,
 
   _getInternationalizedUnit(bytesAmount, language) {
     return formatBytesParts(bytesAmount, language).unit;
```

Comment:
Perhaps check the language here.

Reply:
I would prefer to demand that `language` is always valid.